<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="btn_OpenDir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/OpenDir.png</normaloff>Icon/OpenDir.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Dir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Dir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SaveDir">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/saveDir.png</normaloff>Icon/saveDir.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_NextImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/right-arrow.png</normaloff>Icon/right-arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>370</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre Image</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_PreImage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/left_arrow.png</normaloff>Icon/left_arrow.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SaveRect">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/Save-icon.png</normaloff>Icon/Save-icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Img">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>651</width>
     <height>431</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(255, 255, 255, 255), stop:0.373979 rgba(255, 255, 255, 255), stop:0.373991 rgba(33, 30, 255, 255), stop:0.624018 rgba(33, 30, 255, 255), stop:0.624043 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_currentImage">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>550</y>
     <width>641</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Posittion">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>20</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x,y,w,h</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Yolo">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>format</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Posittion</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_labels">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>380</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Directory Image:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dirImage">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>410</y>
     <width>271</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>------------------------</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_dirLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>520</y>
     <width>261</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>------------------------</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>490</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Directory label:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalImage">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>440</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>550</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_totalImage_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>440</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Total :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_removeLabel">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>320</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/remove__.png</normaloff>Icon/remove__.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clearLabel">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>320</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icon/clear.png</normaloff>Icon/clear.png</iconset>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_Image">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>460</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
